### PR TITLE
Add no-config CheckboxQuestionComponent

### PR DESCRIPTION
**Pull request by mellogarrett**
## Jira Issue

n/a

## Description

- Create "no-config" version of the `<CheckboxQuestionComponent/>`
- In "no-config" `<FormsComponent/>`, add margin between questions

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: no-config-components/dropdown-question/index.tsx [hunk 1]
```diff
@@ -19,11 +19,10 @@ import { useForm } from '../form';
 
 export interface DropdownQuestion {
   title: string;
+  description?: string;
   isRequired: boolean;
   key: QuestionKey;
   choices: Choice[];
-  description?: string;
-  prepopulateKey?: string;
```
Comment (MarkDKoslow) [inline no-config-components/dropdown-question/index.tsx]: Should this have been removed?

File: no-config-components/form/index.tsx [hunk 2]
```diff
@@ -109,6 +109,10 @@ const generateStyles: StyleGenerator = () => {
     formBodyStyles: css`
       ${responsiveWidthSnippet}
       margin: 16px auto 64px;
+
+      & > * {
```
Comment (MarkDKoslow) [inline no-config-components/form/index.tsx]: Does this mean these styles would get applied to all elements?

Comment (mellogarrett) [reply no-config-components/form/index.tsx]: Just the direct child elements, so the container divs for each question type component

File: no-config-components/dropdown-question/index.tsx (hunk 1 above)
Comment (mellogarrett) [reply no-config-components/dropdown-question/index.tsx]: Yea, we won't prepopulate the same way we do now. We'd instead fetch some data in the top-level form component and pass it into this component props. Something like:

```ts
const Form = () => {
  const choices = atlas.getChoices();

  return (
    <FormComponent title={'No Config' as any}>
      <CheckboxQuestionComponent
        question={{
          title: 'Checkbox',
          choices,
          isRequired: false,
          key: 'test-checkbox' as any,
        }}
      ></CheckboxQuestionComponent>
    </FormComponent>
  );
};
```

**Pull request merged**